datahub-web-react/src/app/ingest/source/IngestionSourceList.tsx

```diff
@@ -197,6 +200,19 @@ export const IngestionSourceList = () => {
                 });
         } else {
             // Create
+            const newSource = {
+                urn: PLACEHOLDER_URN,
```

Comment:
So we still do the 3 second timeout? To refetch. I bet if we return the URN from the createSource call we wouldn't even need that followup